src/Farmer/Builders/Builders.WebApp.fs

```diff
@@ -603,7 +605,8 @@ type WebAppBuilder() =
           AutomaticLoggingExtension = true
           SiteExtensions = Set.empty
           PrivateEndpoints = Set.empty
-          CustomDomains = Map.empty }
+          CustomDomains = Map.empty
+          ZoneRedundant = None }
```

Comment:
Should `None` be the default or `false`? What happens if you build a server farm without setting `enable_zone_redundant`?

Reply:
Farmer already had zone redundancy for EventHub, there it was kinda nullable. Tried to keep it consistent. 
When we don't specify it, farmer doesn't include ZoneRedundant property in ARM template and by default Azure doesn't make it ZoneRedundant. I will give a test with ZoneRedundant false in ARM template and let you know

Reply:
Happy to keep it consistent with EventHub, I just wanted to make sure we haven't broken anything that doesn't specify zone redundancy :)